apps/condo/domains/common/utils/date.js

```diff
@@ -10,7 +10,7 @@ const DATE_FORMAT_Z = 'YYYY-MM-DDTHH:mm:ss.SSSZZ'
  * @param shouldFormat
  * @returns {*}
  */
-const getMonthStart = (dateRaw, shouldFormat = false) => {
+const getMonthStart = (dateRaw = undefined, shouldFormat = false) => {
```

Comment:
Please add test cases for dateRaw=undefined

Reply:
Test already exists. Function is used without arguments and there are warnings about it